src/pages/about.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import styled from 'styled-components'
+
+import dotGridPaper from '../images/dot-grid.png'
+
+const MainWrapper = styled.div`
+  background-image: url(${dotGridPaper});
+  background-repeat: repeat;
+  display: flex;
+`
+
+const PageTitle = styled.h1`
+  color: #f3c39a;
+`
+
+const style = {
+  textDecoration: 'none',
+  marginTop: '20px',
+}
+
+class AboutPage extends React.Component {
```

Comment:
What do you think about making this a functional component? Looks like `AboutPage` is not using an internal state anyway.